txscript/standard_test.go

```diff
@@ -671,27 +698,52 @@ func TestMultiSigScript(t *testing.T) {
 	t.Parallel()
 
 	//  mainnet p2pk 13CG6SJ3yHUXo4Cr2RY4THLLJrNFuG3gUg
-	p2pkCompressedMain, err := dcrutil.NewAddressSecpPubKey(decodeHex("02192d7"+
```

Comment:
Please don't change this. There is no need since they are specifically being given the compressed bytes.